dev/io.openliberty.microprofile.health.internal.common/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -17,7 +17,8 @@
                    localization="OSGI-INF/l10n/metatype">
 
     <OCD id="io.openliberty.microprofile.health" name="%mpHealth" description="%mpHealth.desc" ibm:alias="mpHealth" ibm:beta="true">
-        <AD id="fileUpdateInterval" name="%fileUpdateInterval" description="%fileUpdateInterval.desc" required="false" type="String" ibm:beta="true"/>
+        <AD id="checkInterval" name="%checkInterval" description="%checkInterval.desc" required="false" type="String" ibm:beta="true"/>
```

Comment:
Would be good to set the `ibm:type="duration(ms)"` here, also the default `default="100s"` value

Reply:
There is no default value for `checkInterval`. The use of duration with its current localized support could lead to scenarios where someone types in _invalid_ values would be recognized as a time unit of some sort. THe direction is that we want to support the "standard" time units.